src/fontMetrics.js

```diff
@@ -55,49 +53,28 @@ const sigmas = {
     delim1: [2.390, 1.700, 1.980],      // sigma20
     delim2: [1.010, 1.157, 1.420],      // sigma21
     axisHeight: [0.250, 0.250, 0.250],  // sigma22
-};
-
-// These font metrics are extracted from TeX by using
-// \font\a=cmex10
-// \showthe\fontdimenX\a
-// where X is the corresponding variable number. These correspond to the font
-// parameters of the extension fonts (family 3). See the TeXbook, page 441.
-const xi1 = 0;
-const xi2 = 0;
-const xi3 = 0;
-const xi4 = 0;
-const xi5 = 0.431;
-const xi6 = 1;
-const xi7 = 0;
-const xi8 = 0.04;
-const xi9 = 0.111;
-const xi10 = 0.166;
-const xi11 = 0.2;
-const xi12 = 0.6;
-const xi13 = 0.1;
 
-// This value determines how large a pt is, for metrics which are defined in
-// terms of pts.
-// This value is also used in katex.less; if you change it make sure the values
-// match.
-const ptPerEm = 10.0;
+    // These font metrics are extracted from TeX by using tftopl on cmex10.tfm;
+    // they correspond to the font parameters of the extension fonts (family 3).
+    // See the TeXbook, page 441. In AMSTeX, the extension fonts scale; to
+    // match cmex7, we'd use cmex7.tfm values for script and scriptscript
+    // values.
```

Comment:
I'm surprised that the third value in each of the arrays isn't determined by `cmex5` similar to how `cmsy5` is used for the values above.